edu.wpi.first.wpilib.plugins.cpp/src/main/java/edu/wpi/first/wpilib/plugins/cpp/wizards/file_template/FileTemplateWizard.java

```diff
@@ -105,7 +105,7 @@ private void doFinish(IProject project, String className, String folderName, IPr
 			url = new URL(WPILibCPPPlugin.getDefault().getBundle().getEntry("/resources/templates/"), source+".cpp");
 			ProjectCreationUtils.createTemplateFile(project, filepath+".cpp", url, map);
 		} catch (MalformedURLException e) {
-            WPILibCPPPlugin.logError("Malforemd URL: "+WPILibCPPPlugin.getDefault().getBundle().getEntry("/resources/templates/")+"/"+source+".h", e);
+            WPILibCPPPlugin.logError("Malformed URL: "+WPILibCPPPlugin.getDefault().getBundle().getEntry("/resources/templates/")+"/"+source+".h", e);
```

Comment:
Can we fix the indentation since you're touching this line?  Should use tabs for consistency with the rest of the file?